lib/facter/mysqld_version.rb

```diff
@@ -0,0 +1,5 @@
+Facter.add("mysqld_version") do
+  setcode do
+    Facter::Util::Resolution.exec('mysqld -V')
```

Comment:
Does this one not need the `.compact` method chained on it, like it used to do on the provider?